docs/kyma-environment-broker/dictionary.md

```diff
@@ -0,0 +1,25 @@
+# Dictionary 
```

Comment:
I'm not sure whether this document should be put on the open-source GitHub as it contains many SAP-related words.